src/utils/dataAttributes/parser.ts

```diff
@@ -61,13 +62,14 @@ export function parseAddButtonDataAttributes(target: HTMLElement): ParseResult {
   const position = target.getAttribute(DataAttribute.AddButtonInsertPosition);
   const pattern = getAddButtonDataAttributesPattern(position);
 
-  logDebug('Parsing add button data attributes for element: ', target);
-  logDebug('Parsing values from data attributes following this pattern: ', pattern);
+  logDebug("Parsing add button data attributes for element: ", target);
+  logDebug("Parsing values from data attributes following this pattern: ", pattern);
 
   return parseDataAttributes(pattern, target);
 }
 
 function getEditButtonDataAttributesPattern(type: HighlightType): DataAttributesParserPattern {
+  // eslint-disable-next-line @typescript-eslint/switch-exhaustiveness-check
```

Comment:
I'd prefer to replace the `default` with a case for the none value. Then there would be a check that no case is missed.